WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -114,7 +114,7 @@ public static void viewBlogAdmin(Context context, Blog blog) {
         slideInFromRight(context, intent);
     }
 
-    public static void addNewBlogPostOrPage(Context context, Blog blog, boolean isPage) {
+    public static void addNewBlogPostOrPage(Activity context, Blog blog, boolean isPage) {
```

Comment:
For consistency with the other routines, this should be renamed `addNewBlogPostOrPageForResult`.